lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -0,0 +1,32 @@
+class MulticastLogger < Logger
+  attr_accessor :loggers
+
+  def initialize(*loggers)
+    require 'set'
+    @loggers = Set.new
```

Comment:
On that note, since you aren't doing any set notation, nor lookups, it's much more efficient to just use a normal Array.